### PR TITLE
Created base view, close #3

### LINKED ISSUE #3: Create Base View
`index.html`

### PR DESCRIPTION
**Pull request by ganneo**
Completed #3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,9 @@ dist
 
 # TernJS port file
 .tern-port
+
+# local development DB
+db
+
+# vscode editor
+.vscode
```
Comment (yugan2005) [inline .gitignore]: Add a new line after this line

File: package.json [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
   "version": "1.0.0",
-  "description": "Awesomememo frontend",
+  "description": "Front end of awesomememo",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --output-public-path=/build/  --mode development --progress --open --hot",
+    "build": "webpack --mode development --progress",
+    "json-server": "json-server --watch ./db/db.json"
```
Comment (yugan2005) [inline package.json]: You can combine this line with Line 8 and run in one command

**Review by yugan2005 — CHANGES_REQUESTED** [review]

**Comment by ganneo** [comment]
@yugan2005  Comment addressed

**Comment by yugan2005** [comment]
LGTM

**Pull request merged**